src/Icicle/Source/ToCore/Prim.hs

```diff
@@ -197,6 +197,15 @@ convertPrim p ann resT xts
   gofun DaysEpoch
    = return $ primmin $ Min.PrimDateTime Min.PrimDateTimeDaysEpoch
 
+  gofun Seq
```

Comment:
actually!
you can actually just convert
``seq a b`` to ``b``.
this is because the reify pass will have already added the case statement to unwrap "a" if it is a Possibly. and if b is a possibly, it will just unwrap it and rewrap it.
yes - conversion for seq is actually trivial.